### DISCUSSION TITLE
Multiple audio in/out

### DISCUSSION THREAD

**Discussion by alfonso73** [DISTRHO/Cardinal#198] [General]
Hi all, Cardinal is coming along very nicely! I wonder if the choice to have stereo in/out for plain audio in plugin version has been made for licensing reason related to VCV or it's just a design choice. I think that multiple outs are super useful i.e. for drums patch so every kit piece can go to its own individual audio. Multiple  ins are super useful for vocoding, sidechain processes, ring modulation etc.

**Comment by falkTX** [comment]
Have you seen the readme? the multiple variants are explained there.
Use LV2 or VST3 and the main variant for the multi IO

**Comment by alfonso73** [comment]
Thanks @falkTX i didn't actually seen the VST3 variant specs! Fine!
